src/config/dynamic-theme-fixes.config

```diff
@@ -9722,6 +9722,19 @@ IGNORE INLINE STYLE
 
 ================================
 
+inet.se
+
+INVERT
+.h1yh8nho
```

Comment:
How can I find this element? This seems to be hidden by default.

Reply:
It's the compact menu selector that only appears when site is in a narrow window to back or show/hide main categories. Same issue here as with icons tha it's black by default and hard to see